packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -150,19 +147,12 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
     _tabController?.dispose();
     _tabController = TabController(length: widget.tabs.length, vsync: this);
 
-    _handleScreenChange(widget.tabs[_tabController.index]);
+    _currentScreen = ValueNotifier(widget.tabs[_tabController.index]);
```

Comment:
nit: to avoid bugs in general prefer to initialize ValueNotifiers in the constructor and make the field final. That way there are never hard to track down issues due to multiple valueNotifiers for the same field.

Reply:
Sounds good - will change to do that.